backends/platform/sdl/switch/switch.cpp

```diff
@@ -164,3 +166,29 @@ Common::HardwareInputSet *OSystem_Switch::getHardwareInputSet() {
 
 	return inputSet;
 }
+
+Common::String OSystem_Switch::getSystemLanguage() const {
+	u64 lang;
+	SetLanguage langcode;
+
+	setInitialize();
+	setGetSystemLanguage(&lang);
+	setMakeLanguage(lang, &langcode);
+
+	switch (langcode) {
+		case SetLanguage_JA:   return "ja_JP";
+		case SetLanguage_ENUS: return "en_US";
+		case SetLanguage_FR:   return "fr_FR";
+		case SetLanguage_FRCA: return "fr_FR";
```

Comment:
Why fr_FR and not fr_CA?

Reply:
fr_CA doesn't exist in LanguageDescription.